files/es/web/html/attributes/min/index.md

```diff
@@ -0,0 +1,88 @@
+| {{HTMLElement("input/date", "date")}}                             | `yyyy-mm-dd`                      | `<input type="date" min="2019-12-25" step="1">`        |
+| {{HTMLElement("input/month", "month")}}                         | `yyyy-mm`                         | `<input type="month" min="2019-12" step="12">`         |
+| {{HTMLElement("input/week", "week")}}                             | `yyyy-W##`                        | `<input type="week" min="2019-W23" step="">`           |
+| {{HTMLElement("input/time", "time")}}                             | `hh:mm`                           | `<input type="time" min="09:00" step="900">`           |
+| {{HTMLElement("input/datetime-local", "datetime-local")}} | `yyyy-mm-ddThh:mm`                | `<input type="datetime-local" min="2019-12-25T19:30">` |
+| {{HTMLElement("input/number", "number")}}                     | [number](/es/docs/Web/CSS/number) | `<input type="number" min="0" step="5" max="100">`     |
+| {{HTMLElement("input/range", "range")}}                         | [number](/es/docs/Web/CSS/number) | `<input type="range" min="60" step="5" max="100">`     |
+
+> **Nota:** Cuando los datos ingresados por el usuario no se adhieren al valor mínimo establecido, el valor se considera inválido en la restricción de validación y coincidirá con la pseudoclase `:invalid`
+
+Consulta [Validación del lado del cliente](/es/docs/Web/Guide/HTML/HTML5/Constraint_validation) y {{DOMxRef("ValidityState.rangeUnderflow", "rangeUnderflow")}} para más información.
+
+Para el elemento {{HTMLElement('meter')}}, el atributo {{HTMLAttrxRef("min", "input")}} define el límite numérico inferior del rango medido. Debe ser menor que el valor mínimo (atributo [max](/es/docs/Web/HTML/Attributes/max), si se especifica. En ambos casos, si se omite, el valor predeterminado es 1.
+
+| Tipo del `input`                 | Sintaxis                          | Ejemplo                                                                                            |
+| -------------------------------- | --------------------------------- | -------------------------------------------------------------------------------------------------- |
+| {{HTMLElement("meter")}} | [number](/es/docs/Web/CSS/number) | `<meter id="fuel" min="0" max="100" low="33" high="66" optimum="80" value="40"> at 40/100</meter>` |
+
+### Impacto en `step`
+
+Los valores de {{HTMLAttrxRef("min", "input")}} y `step` definen cuáles son los valores válidos, incluso si el atributo `step` no está incluido, como el `step` predeterminado de `0`.
+
+Agrega un gran borde rojo alrededor de los {{HTMLElement("input")}}s no válidos:
+
+```css
+input:invalid {
+  border: solid red 3px;
+}
+```
+
+Luego define un {{HTMLElement("input")}} con un valor mínimo de 7.2, omitiendo el atributo `step`, en donde el valor predeterminado es 1.
+
+```
+<input id="myNumber" name="myNumber" type="number" min="7.2" value="8">
+```
```

Comment:
to fix the embedlivesample

````suggestion
```html
<input id="myNumber" name="myNumber" type="number" min="7.2" value="8">
```
````